src/GitHub.App/Services/GitClient.cs

```diff
@@ -14,6 +14,7 @@ namespace GitHub.Services
 {
     [Export(typeof(IGitClient))]
     [PartCreationPolicy(CreationPolicy.Shared)]
+    [NullGuard(ValidationFlags.None)]
```

Comment:
Leave it on there, `NullGuard` being enabled causes more problems than it solves IMO and we will hopefully remove it soon.